TYPO3.Flow/Classes/TYPO3/Flow/Reflection/ObjectAccess.php

```diff
@@ -245,6 +246,30 @@ public static function setProperty(&$subject, $propertyName, $propertyValue, $fo
             } else {
                 $subject->$propertyName = $propertyValue;
             }
+        } elseif (self::isCollectionTypePropertyValue($propertyValue)
+                  && is_callable(array($subject, $addMethodName = self::buildAdderMethodName($propertyName)))
+                  && is_callable(array($subject, $removeMethodName = self::buildRemoverMethodName($propertyName)))) {
+            $itemsToAdd = is_object($propertyValue)?iterator_to_array($propertyValue):(array)$propertyValue;
```

Comment:
Interesting, that is not regulated in PSR-2? I would expect spaces between the ternary operators.
Also strictly speaking `is_object($propertyValue)` should probably be `($propertyValue instanceof \Iterator)` but I guess it's fine.

Same in line 255.